tool/README.md

```diff
@@ -26,8 +26,9 @@ In order to build the above-mentioned benchmarking tools, absolute paths to each
 | ------------- | ------------- |
 | awslc_bm | -DAWSLC_INSTALL_DIR |
 | bssl_bm | -DBORINGSSL_INSTALL_DIR |
-| ossl_bm | -DOPENSSL_INSTALL_DIR |
-| ossl3_bm | -DOPENSSL3_INSTALL_DIR |
+| ossl_1_0_bm | -DOPENSSL_1_0_INSTALL_DIR |
```

Comment:
Use full version numbers? There are multiple other versions than `1.0.2` and `1.1.1`.

Reply:
I think this should be compatible with any 1.0.x/1.1.x but I'll do some work to double check that

Reply:
Verified that the `1.0` version works with the stable releases of OpenSSL `1.0.0` `1.0.1` and `1.0.2` and the `1.1` version works with both `1.1.0` and `1.1.1`. I haven't looked into any of the `0.x` releases but I doubt anyone is going to need to compare those versions performance to aws-lc's performance.